<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>screenwriiter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="plus_button">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>266</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="minus_button">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>266</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QToolButton" name="compose_button">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>266</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Compose</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_box">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>362</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>TYPE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SLGL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SUBH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ACTN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHAR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DLOG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PNTL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXTN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRSN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SHOT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="text_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>360</width>
      <height>240</height>
     </rect>
    </property>
    <property name="plainText">
     <string>TEXT</string>
    </property>
   </widget>
   <widget class="QListWidget" name="blocks">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>11</y>
      <width>210</width>
      <height>245</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save_as"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Recent">
   <property name="text">
    <string>&amp;Recent</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="text">
    <string>&amp;Save as..</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About..</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
